x-pack/metricbeat/module/sql/_meta/fields.yml

```diff
@@ -1,6 +1,5 @@
 - key: sql
   title: "SQL"
-  release: beta
```

Comment:
Removing the release tag doesn't seems to work in Beats. Can we add release tag with `ga`.